simtools/schemas/model_parameter_and_data_schema.metaschema.yml

```diff
@@ -98,6 +98,11 @@ definitions:
           Use `file` if the parameter describes a file name.
           Use `data_table` if the parameter describes a data table
           (requires TABLE_COLUMNS description)"
+      json_schema:
+        $ref: "http://json-schema.org/draft-06/schema#"
```

Comment:
Previously we used draft 7 and here it is draft 6. Does it matter?

Reply:
No, we always had draft 6 (the format checker for astropy units was the only place where we mentioned Draft7.

I honestly don't know enough about the differences.

Reply:
OK, before I saw something about Draft7, but it doesn't matter.

Reply:
It was here: https://github.com/gammasim/simtools/blob/1c3acd9c1e7498214527db7d3045aa2447804e5d/simtools/data_model/metadata_model.py#L24

But this line has been changed now that it is independent of the draft. Really not sure why we didn't do it like that from the beginning.

Reply:
OK!